templates/default/engage-nginx-proxy-conf.erb

```diff
@@ -26,8 +33,12 @@ server {
     alias <%= @shared_storage_root %>/downloads;
   }
 
+  location /usertracking {
+    proxy_pass http://127.0.0.1:<%= @matterhorn_backend_http_port %>/usertracking;
+    access_log /var/log/nginx/session_uid.log session_uid;
```

Comment:
Turns out the answer was yes. Updated to repeat the base config's standard access_log setting.